src/interfaces/IPriceOracleAdapter.sol

```diff
@@ -10,13 +10,13 @@ import "solmate/tokens/ERC20.sol";
  * various price oracles. An external price oracle can be used seamlessly
  * by being wrapped in a contract implementing this interface.
  */
-interface IPriceOracle {
+interface IPriceOracleAdapter {
     /**
      * @notice
      * @param token The ERC20 token to retrieve the USD price for
      * @return price The price of the token in USD
      */
-    function getPriceUSD(address token) external view returns (uint256 price);
+    function getPriceUSD(address token) external view returns (int256 price);
```

Comment:
I wouldn't think it could ever have a negative price. `uint256` seems right here, no?

Reply:
agree, may have just missed the `u`